images/sdk/configs/latest.apko.yaml

```diff
@@ -6,7 +6,6 @@ contents:
     - '@local /github/workspace/packages'
   packages:
     - sdk@local
-    - wolfi-baselayout
```

Comment:
Why remove this line? (I had thought we want to include this line in all images)

Reply:
I simply pushed it into the melange file as a dependency of sdk